lib/client.js

```diff
@@ -132,6 +135,14 @@ Client.prototype.getJwtFromUrl = function () {
 };
 
 /**
+ * Attempts to replace the JWT in the url with the new JWT passed in the function.
+ * @param {string} newJwt
+ */
+Client.prototype.replaceJWTInUrl = function (newJwt) {
+  window.location.hash = '/?jwt=' + newJwt;
```

Comment:
The secondary jwt is only good for five minutes, there are other edge cases
to consider here.
On May 6, 2016 08:15, "Timothy E. Johansson" <notifications@github.com>
wrote:

> In lib/client.js
> <https://github.com/stormpath/stormpath.js/pull/93#discussion_r62334970>:
>
> > @@ -132,6 +135,14 @@ Client.prototype.getJwtFromUrl = function () {
> >  };
> >
> >  /**
> > + * Attempts to replace the JWT in the url with the new JWT passed in the function.
> > + * @param {string} newJwt
> > + */
> > +Client.prototype.replaceJWTInUrl = function (newJwt) {
> > +  window.location.hash = '/?jwt=' + newJwt;
>
> @edjiang <https://github.com/edjiang> Oh, I didn't know this was only
> used for ID Site. Then I guess it's fine :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/stormpath/stormpath.js/pull/93/files/35c20f2578fb7b6bb0e2ccb801f69f28fb3dc7e2#r62334970>
>